stdlib/LinearAlgebra/src/lbt.jl

```diff
@@ -103,6 +103,42 @@ struct LBTConfig
     end
 end
 
+Base.show(io::IO, lbt::LBTLibraryInfo) = print(io, "LBTLibraryInfo(", basename(lbt.libname), ", ", lbt.interface, ")")
+function Base.show(io::IO, mime::MIME{Symbol("text/plain")}, lbt::LBTLibraryInfo)
+    summary(io, lbt); println(io)
+    println(io, "├ Library: ", basename(lbt.libname))
+    println(io, "├ Interface: ", lbt.interface)
+      print(io, "└ F2C: ", lbt.f2c)
+end
+
+function Base.show(io::IO, lbt::LBTConfig)
+    if length(lbt.loaded_libs) <= 3
+        print(io, "LBTConfig(")
+        gen = (string("[", uppercase(string(l.interface)), "] ",
+            basename(l.libname)) for l in lbt.loaded_libs)
+        print(io, join(gen, ", "))
+        print(io, ")")
+    else
+        print(io, "LBTConfig(...)")
+    end
+end
+function Base.show(io::IO, mime::MIME{Symbol("text/plain")}, lbt::LBTConfig)
+    summary(io, lbt); println(io)
+    println(io, "Libraries: ")
+    for (i,l) in enumerate(lbt.loaded_libs)
+        char = i == length(lbt.loaded_libs) ? "└" : "├"
+        interface_str = if l.interface == :ilp64
+            "ILP64"
+        elseif l.interface == :lp64
+            " LP64"
+        else
+            "UNKWN"
```

Comment:
This shortening is maybe a bit unusual? Trying to google "UNKWN" I only get references to some artist (https://soundcloud.com/ITSUNKWN). Not too bad music...

Reply:
First of all, I guess this is not supposed to occur anyways, right?

The thing is we have the fallback interface `:unknown` but using "UNKOWN" breaks the formatting. We could, of course, space pad LP64 and ILP64 up to 6 characters but this seems suboptimal.

Would you prefer representing `:unknown` as "?????" or do you have a different suggestion?

Reply:
Seems like the general abbreviation of unknown is "unk" (https://writingexplained.org/english-abbreviations/unknown). Personally, I prefer unknwn over unk, but I don't care much.

Reply:
I guess it doesn't matter much, was mostly a drive-by comment while looking through this.

Reply:
I did think it was a bit strange too, but figured it is almost never going to be shown.